src/components/Traceroute/Traceroute.tsx

```diff
@@ -1,52 +1,40 @@
 import { Button, NativeSelect, Stack, TextInput } from "@mantine/core";
 import { useForm } from "@mantine/form";
-import { useCallback, useState } from "react";
+import { useCallback, useEffect, useState } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
 import { UserGuide } from "../UserGuide/UserGuide";
```

Comment:
You still need to upgrade the component to use RenderComponent as userguide is deprecated. 
```suggestion
import { RenderComponent } from "../UserGuide/UserGuide";
```

Reply:
I have removed User guide and I have updated the code to use the Render Component.